templates/templates/preview/activeinfra_plain_text.template

```diff
@@ -0,0 +1,9 @@
+
+    = Database Identifier: {{ db.entity.db_identifier }} =
```

Comment:
@ankona Chris and I talked about this. I don't mind either way. Do you have a preference Chris?

Reply:
I'm ok with any sort of standard. I like Matt's idea... If it's all plain text in a directory, that would be nice because it lets us see a "complete set" at a glance (and thus, know needs to implement a complete set in another format easily)... it also turns a version switch into a single replacement of a path segment, and that's nice. I'm on board for sure